tests/mocks/services.py

```diff
@@ -1195,6 +1197,9 @@ def __init__(
         self.available = available
         self.location_auth = location_auth
         self.robot_part_history = robot_part_history
+        self.fragment_history = fragment_history
+        self.page_token = ""
+        self.page_limit = 0
```

Comment:
(minor, opt) I'm pretty sure these aren't necessary/slightly surprised to hear you say you had errors if they weren't included. You _should_ be able to delete them, see e.g. how `self.public` is set in the `UpdateFragment` method, despite no `public` field ever being defined prior. 

That said, it's pretty harmless to have this here so it's not a big deal if you don't want to bother figuring out how to remove them.

Reply:
By access, I guess meant "set". But it must have been unrelated because I was able to get rid of those default values successfully, so thank you!
Once tests pass I will merge- thank you for your help on this!!